plugins/modules/win_get_url.ps1

```diff
@@ -264,14 +285,14 @@ if ($force -or -not (Test-Path -LiteralPath $dest)) {
     # force=yes or dest does not exist, download the file
     # Note: Invoke-DownloadFile will compare the checksums internally if dest exists
     Invoke-DownloadFile -Module $module -Uri $url -Dest $dest -Checksum $checksum `
-        -ChecksumAlgorithm $checksum_algorithm
+        -ChecksumAlgorithm $checksum_algorithm -SetZone $set_zone
```

Comment:
When using a switch we can do `-SetZone:$set_zone` to set it based on the module option.